app/config.go

```diff
@@ -284,10 +297,12 @@ func InitTendermint() *node.Node {
 		return NewPocketCoreApp(logger, db, baseapp.SetPruning(store.PruneNothing))
 	})
 	if err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(1)
 	}
 	if err := tmNode.Start(); err != nil {
-		panic(err)
+		fmt.Println(err)
+		os.Exit(1)
```

Comment:
```suggestion
		log.Fatal(err)
```
stdlib package `log` offers `log.Fatalf` which logs the error with a timestamp & calls onto os.Exit(1)

the `log` package needs to also be added on this file for this suggestion to work